state-chain/custom-rpc/src/lib.rs

```diff
@@ -456,6 +456,9 @@ pub trait CustomApi {
 		to_asset: RpcAsset,
 		at: Option<state_chain_runtime::Hash>,
 	) -> RpcResult<Option<Vec<AssetAmount>>>;
+
+	#[method(name = "supported_assets")]
+	fn cf_supported_assets(&self) -> RpcResult<HashMap<ForeignChain, Vec<Asset>>>;
```

Comment:
Please make sure this works at runtime - sometimes serde fails to serialize HashMaps (I think it's ok in this case because ForeignChain serializes to json String, but best to be sure).